### PR TITLE
Withholds PHI fields and logs bulk downloads

### LINKED ISSUE #646: Add user access logging to resolver

### PR DESCRIPTION
**Pull request by hkethi002**
Closes #646

Ref: <https://github.com/flywheel-io/flywheel/issues/462>

### Breaking Changes:
  - Items being removed from list view:
    - session
      - subject.sex
      - subject.age
      - subject.race
      - subject.ethnicity
      - subject.info
      - files.info
      - tags
    - acquisitions
      - files.info
      - tags

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

File: api/handlers/containerhandler.py [hunk 1]
```diff
@@ -67,7 +67,10 @@ class ContainerHandler(base.RequestHandler):
             'storage_schema_file': 'session.json',
             'payload_schema_file': 'session.json',
             # Remove subject first/last from list view to better log access to this information
-            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0, 'subject.lastname': 0},
+            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0,
+                                'subject.lastname': 0, 'subject.sex': 0, 'subject.age': 0,
+                                'subject.race': 0, 'subject.ethnicity': 0, 'subject.info': 0,
+                                'file.info': 0, 'session.tag': 0, 'acquisition.tag': 0},
```
Comment (nagem) [inline api/handlers/containerhandler.py]: This is the list projection specific to a session object mongo object, so `tags` would be the key. 
`file.info` should be `files.info`

File: api/handlers/containerhandler.py [hunk 2]
```diff
@@ -77,7 +80,7 @@ class ContainerHandler(base.RequestHandler):
             'parent_storage': containerstorage.SessionStorage(),
             'storage_schema_file': 'acquisition.json',
             'payload_schema_file': 'acquisition.json',
-            'list_projection': {'info': 0, 'collections': 0}
+            'list_projection': {'info': 0, 'collections': 0, 'files.info': 0}
```
Comment (nagem) [inline api/handlers/containerhandler.py]: And then you can add `tags` here.

File: api/download.py [hunk 3]
```diff
@@ -76,6 +54,29 @@ def archivestream(ticket):
...
+                        break
+                if filtered:
+                    continue
+            if optional or not f.get('optional', False):
+                filepath = os.path.join(data_path, util.path_from_hash(f['hash']))
+                if os.path.exists(filepath): # silently skip missing files
+                    targets.append((filepath, prefix + '/' + f['name'], cont_name, str(container.get('_id')),f['size']))
+                    total_size += f['size']
+                    total_cnt += 1
+            # self.log_user_access(AccessType.download_file, cont_name=cont_name, cont_id=container.get('_id'))
```
Comment (nagem) [inline api/download.py]: Looks like there are a few commented out lines from development still hanging around.

**Comment by ryansanford** [comment]
@kofalt please confirm if sdk is impacted by breaking changes

**Comment by kofalt** [comment]
Remaining items for the resolver:

* File info fields when provided as children nodes
* Determine at which node levels tags are considered PHI
* Subject name is in use by the CLI

**Comment by kofalt** [comment]
This also breaks the SDK, and will need a companion branch to fix.

**Pull request merged**